src/main/osd/osd_elements.c

```diff
@@ -992,6 +993,21 @@ static void osdElementFlymode(osdElementParms_t *element)
     }
 }
 
+static void osdElementReadyMode(osdElementParms_t *element)
+{
+    if (IS_RC_MODE_ACTIVE(BOXREADY) && !isFlying) {
+        if (calculateThrottlePercent() < 1) {
+            strcpy(element->buff, "READY");
+        } else {
+            isFlying = true;
+        }
+    }
+
+    if (!ARMING_FLAG(ARMED) && isFlying) {
+        isFlying = false;
+    }
```

Comment:
```suggestion
    if (IS_RC_MODE_ACTIVE(BOXREADY) && !IS_RC_MODE_ACTIVE(BOXARM)) {
        if (calculateThrottlePercent() < 1) {
            strcpy(element->buff, "READY");
        }
    }
```

Reply:
`isFlying` state machine ...

Reply:
Yeah you can be ready and armed with throttle at 0 before a race start and user will likely want to display ready